Intersect.Editor/Forms/Editors/frmSpell.cs

```diff
@@ -259,9 +259,9 @@ private void InitLocalization()
             lblY.Text = Strings.Warping.y.ToString("");
             lblWarpDir.Text = Strings.Warping.direction.ToString("");
             cmbDirection.Items.Clear();
-            for (var i = -1; i < 4; i++)
+            for (Direction i = Direction.None; i < (Direction)4; i++)
```

Comment:
If the cast isn't needed, remove

Reply:
https://github.com/AscensionGameDev/Intersect-Engine/pull/1708#discussion_r1125942002